File: librarymanager/librarymanager-be/src/main/java/ru/spring5/elena/librarymanager/api/dto/Author.java
package ru.spring5.elena.librarymanager.api.dto;

import lombok.Builder;
import lombok.Getter;

import java.util.Date;
import java.util.Set;

@Builder
@Getter
public class Author {
    private Long authorId;
    private String firstName; //имя
    private String surname; //фамилия
    private String patronymic; //отчество
    private Date birthdate; //
    private Date deathdate; //
    private String info; //доп. информация
    private String fio;
    private Set<Long> edidionIds;
}

File: librarymanager-api/src/main/kotlin/ru/elenacherev/librarymanager/api/dto/Author.kt
package ru.elenacherev.librarymanager.api.dto

import java.util.Date

data class Author(
    var authorId: Long?,
    var firstName: String,
    var surname: String,
    var patronymic: String?,
    var birthdate: Date?,
    var deathdate: Date?,
    var info: String?,
    var fio: String?,
    var edidionIds: List<Long?>
)